x/accounts/defaults/lockup/clawback.go

```diff
@@ -0,0 +1,392 @@
+package lockup
```

Comment:
can we get some docs on this please. maybe a diagram or two?

Reply:
updated the readme file, gonna add a diagram here soon, just read the mermaidjs docs 😅

Reply:
added both now